packages/core/src/SinglePackageTool.ts

```diff
@@ -0,0 +1,67 @@
+import path from 'path';
+import fs from 'fs-extra';
+
+import { Tool, ToolType, Package, PackageJSON, Packages, InvalidMonorepoError } from './Tool';
+import { expandPackageGlobs } from "./expandPackageGlobs";
+
+export const SinglePackageTool: Tool = {
+    type: 'package',
```

Comment:
This probably should match the value declared in this union:
https://github.com/Thinkmill/manypkg/blob/3191db736c686119f3c3da9971872792bf7b8775/packages/core/src/Tool.ts#L4